### ISSUE TITLE
Add an OSI-approved license

### ISSUE THREAD

**Issue by chenrui333** [rharder/imagesnap#32]
Hi, thanks for creating this great software, unfortunately, I do think the license text is not really helpful. But I am happy to help you out on that. If you really looking for public domain license, I can see two choices can be used:

- https://spdx.org/licenses/CC0-1.0.html
- https://spdx.org/licenses/Unlicense.html

The good thing with those two are they are more formal and recognized in the OSI governance body and easy to help your software distribution and adoption.

If you are looking for permissive software licenses:
- Apache-2.0
- MIT
- BSD-3-Clause

are the good ones.

**Comment by rharder** [comment]
Agreed those are great public domain licenses.  I've released this into the public domain, so that's even more open. =)

**Comment by ryandesign** [comment]
The phrase "public domain" is [legally ambiguous](https://opensource.org/node/878). Selecting a specific license would be unambiguous.
